float64 battery_level
string current_state
float64[] joint_positions
